eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/steps/tasklets/reportCheck/CoordinatesVcfFactory.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.pipeline.steps.tasklets.reportCheck;
+
+import uk.ac.ebi.eva.commons.core.models.VariantCoreFields;
+import uk.ac.ebi.eva.commons.core.models.factories.VariantVcfFactory;
+import uk.ac.ebi.eva.commons.core.models.factories.exception.IncompleteInformationException;
+import uk.ac.ebi.eva.commons.core.models.factories.exception.NonVariantException;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+import uk.ac.ebi.eva.commons.core.models.pipeline.VariantSourceEntry;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Transforms a String (in VCF format) to a list of variants, keeping only the basic fields (coordinates):
+ * chromosome, position, id, reference, alternate.
+ */
+public class CoordinatesVcfFactory extends VariantVcfFactory {
+
+    @Override
+    public List<Variant> create(String fileId, String studyId, String line)
+            throws IllegalArgumentException, NonVariantException, IncompleteInformationException {
+        String[] fields = line.split("\t", 6);
+        String chromosome = getChromosomeWithoutPrefix(fields);
```

Comment:
The methods `getChromosomeWithoutPrefix`, `getPosition`, `getReference`, `getAlternateAlleles` from `VariantVcfFactory` class can't be used here because they are private and that is why we are getting the compilation error.

Methods could be changed to protected in the commons project if they need to be used in the subclass.